core/src/main/scala/org/apache/spark/jdbc/ConnectionUtil.scala

```diff
@@ -62,7 +62,9 @@ object ConnectionUtil {
       case SparkContext.DRIVER_IDENTIFIER => connectionProps.connProps
       case _ => connectionProps.executorConnProps
     }
-    val jdbcOptions = new JDBCOptions(connectionProps.url, "", connProps.asScala.toMap)
+    
+    val jdbcOptions = new JDBCOptions(connectionProps.url +
+      connectionProps.urlSecureSuffix, "", connProps.asScala.toMap)
```

Comment:
Any reason to add a separate "urlSecureSuffix" field and not add to the "url" itself? Alternatively it might be cleaner to put those properties in connProps and executorConnProps.